java/src/org/openqa/selenium/firefox/FirefoxDriver.java

```diff
@@ -265,13 +265,11 @@ private static boolean isLegacy(Capabilities desiredCapabilities) {
 
   @Override
   public String installExtension(Path path) {
-    Require.nonNull("Path", path);
```

Comment:
It's fine to leave these checks in. It'll make the exception come from `FirefoxDriver`, and that's probably clearer to a user.

Reply:
I moved all of these to the implementation for consistency. Should I move them back out of the implementation to the driver classes, etc?

Reply:
added to overridden driver classes